nilrt_snac/_configs/_config_file.py

```diff
@@ -65,3 +82,17 @@ def contains(self, key: str) -> bool:
         Returns: True if the key is found, False otherwise.
         """
         return bool(re.search(key, self._config))
+    
+    def get(self, key: str) -> str:
+        """Get the value of the given key in the configuration file.
+
+        Args: key: RE pattern to search for in the configuration file.
+
+        Returns: The value of the key if found, empty string otherwise.
+        """
```

Comment:
I think the _ConfigFile class is supposed to be more generic than to assume that the config file contains `=`-delimited key-value pairs. You might need to subclass, or just regex search the `self._config` string directly.